apps/storybook/src/DefaultInteractions.stories.tsx

```diff
@@ -1,35 +1,63 @@
-import { DefaultInteractions, ResetZoomButton, VisCanvas } from '@h5web/lib';
+import {
+  DefaultInteractions,
+  HeatmapMesh,
+  mockValues,
+  ResetZoomButton,
+  ScaleType,
+  VisCanvas,
+} from '@h5web/lib';
 import type { DefaultInteractionsConfig } from '@h5web/lib/src/interactions/DefaultInteractions';
 import type { Meta, Story } from '@storybook/react';
 
 import FillHeight from './decorators/FillHeight';
+import { useMockData } from './hooks';
+
+const { twoD } = mockValues;
 
 export const InsideAutoAspectCanvas: Story<DefaultInteractionsConfig> = (
   args
 ) => {
+  const { values, domain } = useMockData(twoD, [20, 41]);
+
   return (
     <VisCanvas
-      abscissaConfig={{ visDomain: [0, 10], showGrid: true }}
-      ordinateConfig={{ visDomain: [0, 10], showGrid: true }}
+      abscissaConfig={{ visDomain: [0, values.shape[1]], showGrid: true }}
+      ordinateConfig={{ visDomain: [0, values.shape[0]], showGrid: true }}
       aspect="auto"
     >
       <DefaultInteractions {...args} />
       <ResetZoomButton />
+
+      <HeatmapMesh
+        values={values}
+        domain={domain}
+        colorMap="Viridis"
+        scaleType={ScaleType.Linear}
+      />
```

Comment:
The select-to-zoom interaction is constrained by the axis domains, which makes it look broken on an empty canvas.